conans/client/cmake.py

```diff
@@ -156,3 +165,48 @@ def runtime(self):
         if runtime:
             return "-DCONAN_LINK_RUNTIME=/%s" % runtime
         return ""
+
+    def configure(self, conan_file, args=None, vars=None, source_dir=None, build_dir=None):
```

Comment:
Thats true, they are CMake definitions, not preprocessor, that is what I wanted to say, my hands wrote a different thing :) I am open to any other suggestions, ``variables`` read too long for a named parameter, but could be.